### ISSUE TITLE
MD028 false positives for alert boxes (like notes, tips, warnings, ...)

### ISSUE THREAD

**Issue by JoKeyser** [DavidAnson/markdownlint#2031]
Hi, thank you for your excellent work.

In VSCode, I'm getting many "MD028/no-blanks-blockquote: Blank line inside blockquote" for cases where multiple alert boxes are written directly below each other.

Here an example; a false positive MD028 appears between these two alert boxes:

```markdown
> [!NOTE]
> This is a note that ends here.

> [!TIP]
> This is a different box directly below.
> This will trigger MD028, but it shouldn't.
```

**Comment by DavidAnson** [comment]
This is not a false positive, this is exactly the scenario that rule is documented to warn about: https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md

If you know this syntax is safe in your environment, you can disable the rule in that file or for your entire project. Alternatively, I believe I recall that an inline HTML comment can be inserted between the two blocks to avoid the warning without changing the rendered output.

**Comment by JoKeyser** [comment]
@DavidAnson , thank you for looking at this.

I somewhat agree that _syntactically_ it's a true positive, however _semantically_ it is a false positive: In my example above, my intent is not to write a single "blockquote", instead I'm writing two different "alert boxes". Unfortunately, this goal shares the same notation involving lines starting with `>`.
As such, I consider this an (admittedly understandable) false positive from the linter's point of view.

Would it be possible to not even consider lines leading with `> [!` to be a  "blockquote"? Because they are not meant to start a blockquote, they start alert boxes.

**Comment by DavidAnson** [comment]
Alert boxes are not part of the Markdown or CommonMark definition, and parser behavior for this scenario is known to vary. If you know your parser/process handles the syntax how you expect, MD028 can/should be disabled.

Note that CommonMark behavior is to create separate blocks, but it does not recognize the custom GitHub Alert syntax:

https://dlaa.me/markdownlint/#%25m%23%20Issue%202031%0A%0A%3E%20%5B!NOTE%5D%0A%3E%20This%20is%20a%20note%20that%20ends%20here.%0A%0A%3E%20%5B!TIP%5D%0A%3E%20This%20is%20a%20different%20box%20directly%20below.%0A

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

**Comment by JoKeyser** [comment]
Ok thank you for the clarification, I was not aware that alert boxes are out of scope of CommonMark.
So I'll close this issue. Sorry for the fuss.
